.gitlab-ci.yml

```diff
@@ -171,3 +173,28 @@ force_release_prod:
     RELEASE_COMMAND: scripts/release.sh staging prod --force
   only:
     - staging
+
+verify_sync_partial:on-schedule:
+  stage: verify_sync
+  only:
+    refs:
+      - schedules
+  script:
+    - FILES_LAST_VERIFIED="$(python scripts/dss-ops.py secrets get --secret-names dcp/dss/${DSS_DEPLOYMENT_STAGE}/files_last_verified | jq -r '.[]')"
+    - BLOBS_LAST_VERIFIED="$(python scripts/dss-ops.py secrets get --secret-names dcp/dss/${DSS_DEPLOYMENT_STAGE}/blobs_last_verified | jq -r '.[]')"
+    - BUNDLE_LAST_VERIFIED="$(python scripts/dss-ops.py secrets get --secret-names dcp/dss/${DSS_DEPLOYMENT_STAGE}/bundles_last_verified | jq -r '.[]')"
+    - bash scripts/verify_consistency.sh files "${FILES_LAST_VERIFIED}" | tee file_verify_log
+    - bash scripts/verify_consistency.sh blobs "${BLOBS_LAST_VERIFIED}" | tee blob_verify_log
+    - bash scripts/verify_consistency.sh bundles "${BUNDLE_LAST_VERIFIED}" | tee bundle_verify_log
+    - test "$(cat *_verify_log | wc -l)" -eq 0  # If any inconsistencies, fail pipeline
+
+verify_sync_full:on-schedule:
+  stage: verify_sync
+  only:
+    refs:
+      - schedules
+  script:
+    - bash scripts/verify_consistency.sh files 0 | tee file_verify_log
```

Comment:
where do these log files go?

Reply:
Is there a way of knowing which verify failed based on this test without running the script again?

Reply:
> where do these log files go?

They don't go anywhere - they're written so that we can see if any discrepancies were logged (see [L200](https://github.com/HumanCellAtlas/data-store/pull/2412/files/f50ec3d2fb5220398323245f020a5642e2e151f0#diff-96edf7a6f008de9e928d04e1ae5e12a5R200)) and are also printed to stdout. If any discrepancies are found, it will be listed in one of those files, and caught by L200 causing a failed build, which in turn will notify someone who can check the build log.

> Is there a way of knowing which verify failed based on this test without running the script again?

Yes, this should be clear from the build log.